curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614796cb8086be482d60e0ac.md

```diff
@@ -16,13 +16,15 @@ Change the font color of all the anchor elements within the list elements to som
 You should use the `li > a` selector.
```

Comment:
Now I wonder, considering that this selector is not compelled anymore, should the hint be changed?

Reply:
It still is an important to solve the problem though.  Campers can add stuff on top of it, but it is still fundamentally an `li > a` selector.